cfgmgr/buffer_headroom_mellanox.lua

```diff
@@ -26,6 +27,7 @@ local port_speed = tonumber(ARGV[1])
 local cable_length = tonumber(string.sub(ARGV[2], 1, -2))
 local port_mtu = tonumber(ARGV[3])
 local gearbox_delay = tonumber(ARGV[4])
+local is_8lane = ARGV[5]
```

Comment:
please rename to 'lane_cnt' and make it an 'int' so that it is generic

Reply:
Fixed.

Reply:
BTW, I changed to lane_count in generic code - buffermgrdyn. The lua plugin is still using is_8lane as it's Mellanox specific.